uniffi_bindgen/src/scaffolding/templates/EnumTemplate.rs

```diff
@@ -2,31 +2,38 @@
 // For each enum declared in the UDL, we assume the caller has provided a corresponding
 // rust `enum`. We provide the traits for sending it across the FFI, which will fail to
 // compile if the provided struct has a different shape to the one declared in the UDL.
+//
+// We define a unit-struct to implement the trait to sidestep Rust's orphan rule (ADR-0006).
```

Comment:
Please consider adding a little more to this comment to help future intrepid debuggers who may be working through the generated Rust code. Suggestion:

"""
We define a unit-struct to implement the trait rather than implementing it directly on the enum.
This sidesteps Rust's orphan rules in the case where the underlying type has been imported from
a separate crate (ADR-0006).
"""